backups/backup-influxdb.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+
+set -x
+
+instance=$1
+shard=$2
+
+[ -z "$instance" ] && echo "Usage: $0 <instance name>" && exit 1
+
+
+export KUBECONFIG=$HOME/.kube/config-${instance}.yaml
+
+kubectl config get-contexts
```

Comment:
Does this line do anything?  Is it just to see if you're in the right context?  If it is then you might want to do a read from stdin before you continue, or a sleep 10 or something to let you break it before moving along.

Reply:
Cool thanks, yes I wanted to make sure we are in the right kubectl context. I'll add a `read` to ask for confirmation there.